### PR TITLE
Add Accessibility to Engineering Fundamentals

### PR DESCRIPTION
**Pull request by shanedjones**
# Add Accessibility to Engineering Fundamentals

## What are you trying to address

This PR addresses issue #825 

Accessibility is a critical engineering fundamental and an area where dev crews need more guidance and support. This PR adds Accessibility as an engineering fundamental and provides dev crews with information on how to incorporate accessibility and inclusive design into their projects.

## Description of new changes

- Added Accessibility to engineering fundamentals list
- Created an Accessibility section
- Removed previous unfinished accessibility page in the UI/UX section
- Updated links to accessibility in other sections to point to the new accessibility content

## For all pull requests

- [X] Link to the issue you are solving (so it gets closed)
- [X] Label the pull request with the appropriate area(s)
- [X] Assign potential reviewers (you may also want to contact them on Teams to ensure timely reviews)
- [ ] Assign the project - Engineering Playbook Backlog
- [X] Assign the pull request to yourself

### CODE REVIEW

File: docs/design/design-reviews/trade-studies/template.md [hunk 1]
```diff
@@ -70,16 +70,13 @@ If applicable, describe the boundaries from which we have to design the solution
 
 #### Accessibility
 
-**Accessibility is never optional**. Microsoft has made a public commitment to always produce accessible appliations. For more information visit the official [Microsoft accessibility site](https://www.microsoft.com/en-us/accessibility) and read the [Accessibility](../../../user-interface-engineering/accessibility.md) page in the User Interface Engineering section.
+**Accessibility is never optional**. Microsoft has made a public commitment to always produce accessible appliations. For more information visit the official [Microsoft accessibility site](https://www.microsoft.com/accessibility) and read the [Accessibility](../../../accessibility/README.md) page.
```
Comment (madossan01) [inline docs/design/design-reviews/trade-studies/template.md:73]: Nit: from appl**ia**tion to application

Comment (shanedjones) [reply docs/design/design-reviews/trade-studies/template.md]: Thanks, fixed.

**Review by madossan01 — APPROVED** [review]

**Review by meulta — APPROVED** [review]
Looks good! Thanks adding this very important topic to the playbook

**Comment by TessFerrandez** [comment]
This was never discussed in the champs team - so I have created an issue to move the content back under UX.
We have many requests like this of moving content to the top level but they need to be coupled with an EF champs group, and this has not been done in this case.

**Pull request merged**